assets/scripts/script.js

```diff
@@ -6,11 +6,16 @@
 const implex = (function () {
   const START_TRANSITION_CLASS = 'start';
   const HEADER_ID = 'header';
+  const defaultText = 'File size up to 10MB';
 
   let sections;
   let sectionsToshow;
   let header;
   let headerItems;
+  let uploadBtn;
+  let customText;
+  let removeBtn;
+  let form;
```

Comment:
Same with this names. You have some app, and such const names are not the best option